guides/v2.3/extension-dev-guide/declarative-schema/index.md

```diff
@@ -33,7 +33,7 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or path is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Markup Language (DML) operations: INSERT, UPDATE.
+* *Revertable data patch* - A patch that can be reverted as a module or path is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Modification Language (DML) operations: INSERT, UPDATE.
```

Comment:
Thanks for contributing @kbendyk!
 
I believe the correct term is Data **Manipulation** Language (DML). If you push that change, I'll merge your PR.

> https://en.wikipedia.org/wiki/Data_manipulation_language